project/MimaExcludes.scala

```diff
@@ -36,6 +36,8 @@ object MimaExcludes {
 
   // Exclude rules for 2.3.x
   lazy val v23excludes = v22excludes ++ Seq(
+    // SPARK-18085: Better History Server scalability for many / large applications
+    ProblemFilters.exclude[IncompatibleResultTypeProblem]("org.apache.spark.status.api.v1.ExecutorSummary.executorLogs"),
```

Comment:
what's going on here? I don't see this PR touch the code of `ExecutorSummary`

Reply:
I removed an import and that changed the type of one of the fields.

Reply:
ah i see: `-import scala.collection.Map`. But is this really necessary to break the compatibility?

Reply:
Well, it doesn't break compatibility because this class is not to be used outside Spark (it's only public so mima can detect breakages in the JSON format, and this doesn't change that).

Leaving the old import makes using this class really awkward from other places that don't have that import.